core/src/main/java/hudson/model/Run.java

```diff
@@ -2330,6 +2372,18 @@ public void setDescription(String description) throws IOException {
         this.description = description;
         save();
     }
+
+    /**
+     * Sets a new summary
+     * @param summary Summary to set. Use {@code null} to remove the summary.
+     * @throws IOException Summary update error
+     * @since TODO
+     */
+    public void setSummary(@CheckForNull String summary) throws IOException {
+        checkPermission(UPDATE);
+        this.description = description;
```

Comment:
Bug, you're setting description instead of summary.

Reply:
heh, not tested :(